tests/Nodejs/14_Getters_Setters.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ok } from "assert";
+import "ts-jest";
+import { Logger } from "../../src/index.js";
+import { mockConsoleLog } from "./helper.js";
+
+class MissingSetter {
+  get test(): string {
+    return "test";
+  }
+}
+
+const missingSetter = {
+  get test(): string {
+    return "test";
+  }
+}
+
+describe("Getters and setters", () => {
+  beforeEach(() => {
+    mockConsoleLog(true, false);
+  });
+  test("[class] should not print getters", (): void => {
```

Comment:
This test passes. I question whether it should print the getters/setters, but at least it's tested for now.